src/updaters/node/package-lock-json.ts

```diff
@@ -34,6 +34,16 @@ export class PackageLockJson extends DefaultUpdater {
     if (parsed.lockfileVersion === 2 || parsed.lockfileVersion === 3) {
       parsed.packages[''].version = this.version.toString();
     }
+    if (this.versionsMap) {
+      for (const [, obj] of Object.entries(parsed.packages)) {
+        for (const [name, ver] of this.versionsMap?.entries()) {
+          if (obj.name === name) {
+            obj.version = ver.toString();
+          }
+        }
```

Comment:
Reading this again, this could be slightly more performent:

```suggestion
        const ver = this.versionsMap.get(name);
        if (ver) {
          obj.version = ver.toString();
        }
```

Reply:
oh ha, thats funny I think I had written the original code assuming what I had there was a list and then instead of just reading it again when I modified it to take the map I just.....left it lol. Will fix that.

Reply:
Fixed